src/ctest.ts

```diff
@@ -599,6 +605,30 @@ export class CTestDriver implements vscode.Disposable {
         return undefined;
     }
 
+    private createTestItemAndSuiteTree(testName: string, testExplorerRoot: vscode.TestItem, initializedTestExplorer: vscode.TestController, uri?: vscode.Uri) : TestAndParentSuite {
```

Comment:
There is still an extra space before the : 

```suggestion
    private createTestItemAndSuiteTree(testName: string, testExplorerRoot: vscode.TestItem, initializedTestExplorer: vscode.TestController, uri?: vscode.Uri): TestAndParentSuite {
```

This is causing build failures.

Reply:
Fixed.